.github/workflows/tests_linux.yml

```diff
@@ -235,6 +271,12 @@ jobs:
           cd main
           python -m pip install -r requirements-dev.txt
 
+      - name: Install Stable PennyLane
+        if: inputs.pennylane-version == 'stable'
+        run: |
+          cd main
+          python -m pip uninstall -y pennylane && python -m pip install -U pennylane
```

Comment:
Do we need to uninstall first?

Reply:
Yes, because we install pennylane@master when we install the dev requirements above.